admin/resources/js/components/ClientProvider.tsx

```diff
@@ -89,10 +87,7 @@ export const ClientProvider: React.FC<{ client?: Client }> = ({
         exchanges: [
           errorExchange({
             onError: (error: CombinedError) => {
```

Comment:
It's not clear to me how this is being used or how this change can be tested.  Could someone help me out?

Reply:
Yes, if you're still seeing "GraphQL: Unauthenticated" toasts then it's still working :)